performance/buildSrc/src/main/kotlin/RunKotlinNativeTask.kt

```diff
@@ -18,6 +20,10 @@ open class RunKotlinNativeTask @Inject constructor(
     var buildType = "RELEASE"
     var workingDir: Any = project.projectDir
     var outputFileName: String? = null
+    @Input
+    @Option(option="subkernel", description="subkernel")
```

Comment:
option ="subkernel"

Reply:
What is subkernel?

Reply:
This is set of separate benchmarks from benchmark application. I don't know what name would be better for this.